src/PharoThings-Devices-HD44780.package/PotLCD1602Device.class/instance/initialize.st

```diff
@@ -0,0 +1,7 @@
+initialization
+initialize
+	super initialize.
+	self
+		displayControl: #('LCD_DISPLAYON' 'LCD_CURSOROFF' 'LCD_BLINKOFF');
+		displayFunction: #('LCD_4BITMODE' 'LCD_1LINE' 'LCD_2LINE' 'LCD_5x8DOTS');
+		displayMode: #('LCD_ENTRYLEFT' 'LCD_ENTRYSHIFTDECREMENT')
```

Comment:
Seems PotLCD1602DeviceI2C and PotLCD2002DeviceI2C are equally configured. And other couple without I2C suffix too.

Reply:
Yes, it is because I was thinking to leave the subclasses to configure different LCD displays. But I don't have them to test and see which parameters could be different.

Reply:
As alternative you can have different class side methods:
```Smalltalk
PotLCD lcd1602I2C
PotLCD lcd2002I2C
```
I wonder why they have suffix I2C but they do not work through I2C connection?
(I see there is another I2C device which do exactly this)

Reply:
I will think about this. For now, I just removed the subclasses. 
The PotLCD1602DeviceI2C is working. This is not a subclass of I2C, but I instantiate an I2C class in an instance variable in PotLCDI2C>>connect to use the protocol by subclasses.